### ISSUE TITLE
a push-up care for ui-exports

### ISSUE THREAD

**Issue by osher** [mochajs/mocha#1144]
Hi, 

I built this [mocha-ui-exports](https://github.com/osher/mocha-ui-exports) for my teams, because I had to get them started and move fast, and did not have time to  figure how you'd like the PR and wait for it to be merged. 
In fact, every past communication attempts with this group wasn't smooth, so I could not afford to wait and promised myself to make a try later (which is now).

However, if anyone can guide me through with the 'rules of the house', - i.e I could close my separate project once the PR is merged :) 

To have it done I had to read mocha in depth, and I find it fascinating to see how it grew in the past years (on our previous evaluation points we preferred vows).

Basically, I need a little help to understand how to fit my tests in your suite, how you want them organized in coherence with the rest of it, and how you want them run (im a spoiled windows user, never used _make_ before, and not too keen to get started, but will do if that's what it takes...).

If anybody would like to communicate with me about it - I'd be more than happy to contribute.
I respond to my work-mail: osher(o)7elementsstudios.com (the one I associated with public sites is a spam filter).

Thanks
    Osher
